sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    ALTER TABLE participants DROP COLUMN bitcoin;
```

Comment:
Are you sure this is right? I made this up (that's what "untested" meant in my [comment](https://github.com/gratipay/gratipay.com/pull/4187#issuecomment-272965641)), and now that I look I don't find a `bitcoin` column in `participants`. I'm actually surprised that Travis is okay with this.